queries/css.scm

```diff
@@ -1,34 +1,140 @@
 ;; https://github.com/tree-sitter/tree-sitter-css/blob/master/src/grammar.json
 
-(string_value) @string @textFragment
+[
+  (at_rule)
+  (charset_statement)
+  (import_statement)
+  (keyframes_statement)
+  (media_statement)
+  (namespace_statement)
+  (rule_set)
+  (supports_statement)
+] @statement
 
-(comment) @comment @textFragment
+;;!! "hello"
+;;!   ^^^^^
+(
+  (string_value) @string @textFragment
+  (#character-range! @textFragment 1 -1)
+)
 
-(declaration) @collectionItem
+;;!! /* Hello */
+;;!  ^^^^^^^^^^^
+(comment) @comment @textFragment
 
+;;!! calc(100% - 50px);
+;;!  ^^^^^^^^^^^^^^^^^
 (call_expression
   (function_name) @functionCallee
 ) @functionCall @functionCallee.domain
 
+;;!! div {}
+;;!  ^^^
 (rule_set
   (selectors) @selector
 ) @_.domain
 
+;;!! width: 100px;
 (declaration
-  (property_name) @name
-) @_.domain
+  (property_name) @name @collectionKey
+  .
+  (_) @value.start
+  (_)? @value.end
+  .
+) @collectionItem @_.domain
+
+;;!! a[target="_blank"]
+;;!    ^^^^^^
 (attribute_selector
   (attribute_name) @name
+  (string_value) @value
 ) @_.domain
 
-(stylesheet) @name.iteration
+;;!! @import "subs.css"
+;;!          ^^^^^^^^^^
+(import_statement
+  (_) @value
+) @_.domain
+
+;;!! translate(-50%, -50%)
+;;!            ^^^^  ^^^^
+(
+  (arguments
+    (_)? @_.leading.endOf
+    .
+    [
+      "("
+      ","
+      (
+        (plain_value) @_dummy
+        (#eq? @_dummy "at")
+      )
+    ]
+    .
+    (_) @argumentOrParameter
+    (
```

Comment:
how come you can't do something like the following instead of adding the new `grow-to-named-siblings!` predicate?

```suggestion
    (_) @argumentOrParameter.start
    (_)? @argumentOrParameter.end
    (#not-eq? argumentOrParameter.end "at")
    .
    (
```

Reply:
I could never get something like that to work. Your example here doesn't work either. I would happily accept a solution like this that works, but I couldn't find it on my own.